src/sentry/conf/server.py

```diff
@@ -1036,11 +1036,10 @@ def SOCIAL_AUTH_DEFAULT_USERNAME():
     "organizations:metrics": False,
     # Enable metric alert charts in email/slack
     "organizations:metric-alert-chartcuterie": False,
-    # Automatically extract metrics during ingestion.
-    #
-    # XXX(ja): DO NOT ENABLE UNTIL THIS NOTICE IS GONE. Relay experiences
-    # gradual slowdown when this is enabled for too many projects.
+    # Extract metrics for sessions during ingestion.
     "organizations:metrics-extraction": False,
+    # Extraction metrics for transactions during ingestion.
+    "organizations:transaction-metrics-extraction": False,
```

Comment:
This was missing entirely. The default was implicitly `False`.

Reply:
Should we remove this flag's fallback to Flagr? Currently it has a `True` for the fallback.

Reply:
👍  It's removed with 3659729.